source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
+        </System>
+        <Integrator ...>
+            <!-- XML content of the OpenMM serialized integrator -->
+        </Integrator>
+    </OpenMMSimulation>
+
+Components Description
+----------------------
+
+1. **Starting Structure**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   * Tag: ``<pdbx>`` (preferred) or ``<pdb>`` (for backward compatibility)
+   * Content: The entire PDBx or PDB file content
+
+2. **OpenMM Serialized System**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Within this tag, you'll find:
+
+    **Particles:** They are represented with ``<Particle>`` tags, each containing:
+        - Mass
+        - Charge (if using a `NonbondedForce`)
+
+    **Forces:** Various force components are represented by specific tags (a detailed list could be found `here <http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#standard-forces>`_), for example:
+
+        - ``<HarmonicBondForce>``: For bond stretching
+        - ``<HarmonicAngleForce>``: For angle bending
+        - ``<PeriodicTorsionForce>``: For dihedral angles
+        - ``<NonbondedForce>``: For electrostatic and van der Waals interactions
+        - ``<CustomNonbondedForce>``: For user-defined nonbonded interactions
+
+        Each force tag contains parameters specific to that force type. For example:
+
+        .. code-block:: xml
+
+           <HarmonicBondForce>
+             <Bond p1="0" p2="1" length="0.1" k="1000"/>
+           </HarmonicBondForce>
+
+    **Constraints:** If present, constraints are listed under a ``<Constraints>`` tag:
+
+        .. code-block:: xml
+
+           <Constraints>
+             <Constraint p1="0" p2="1" distance="0.1"/>
+           </Constraints>
+
+
+3. **OpenMM Serialized Integrator**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``<Integrator>`` tag contains parameters that specify the integration method to be used to simulate dynamics, such as the type of integrator, simulation time step and temperature:
+
+.. code-block:: xml
+
+    <Integrator type="LangevinIntegrator" constraintTolerance="1e-05" friction="4" randomSeed="0" stepSize=".0005" temperature="300" version="1" />
+
+More details on integrators can be found `here <http://docs.openmm.org/latest/userguide/theory/04_integrators.html>`_.
+
+Usage
+-----
+
+The :mod:`nanover.openmm.serializer` module provides the ``serialize_simulation`` and ``deserialize_simulation`` functions which allow saving and loading OpenMM simulations to/from XML files. The serialization captures the complete simulation state including:
+
+- Structure coordinates and topology (as PDBx/PDB)
+- OpenMM System definition
+- Integrator configuration
+
+Serializing a Simulation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To save a simulation to XML::
+
+    xml_string = nanover.openmm.serializer.serialize_simulation(simulation)
+
+    with open("sim.xml", "w") as f:
+        f.write(xml_string)
+
+Deserializing a Simulation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To load a simulation from XML::
```

Comment:
I think it would be good to make it clear that this is loading a simulation from a NanoVer OpenMM XML file, not just a general XML file. (Same comment for below)

Reply:
resolved in commit [f62ca6e](https://github.com/IRL2/nanover-docs/pull/115/commits/f62ca6e5fd4d7a15e0d7843f064081e5d054ccf2)